src/map_field.cpp

```diff
@@ -1585,7 +1588,7 @@ void map::player_in_field( Character &you )
                                 parts_burned.emplace_back( "leg_l" );
                                 parts_burned.emplace_back( "leg_r" );
                         }
-                    } else if( you.has_effect( effect_quadruped_full ) ) {
+                    } else if( you.has_effect( effect_quadruped_full ) ||  you.has_effect( effect_quadruped_half ) ) {
```

Comment:
I'd prefer to remove this too, but nothing comes to mind so I won't insist on it.